blockfish-client/src/controller.rs

```diff
@@ -206,6 +222,52 @@ impl<'v> Controller<'v> {
         self.update_view(upd);
     }
 
+    /// compares the selected move against the suggestions
+    fn evaluate_selected_move(&mut self, an: &mut Analysis) {
+        // get the best move from the analysis
+        let best_rating = an.moves.first().unwrap().rating;
+        let mut best_prior_move = an.clone_as_best();
+        let actual_board = self.stacker.snapshot().unwrap();
+
+        for _ in 0..an.moves.len() {
+            let mv = &an.moves[an.sel_idx];
+            let mut next_suggestion = self.stacker.clone();
+
+            const SUGGESTION_THRESHOLD: i64 = 20;
+            if an.go_to(&mut next_suggestion) {
+                next_suggestion.hard_drop();
+                let next_suggestion = next_suggestion.snapshot().unwrap();
+
+                if next_suggestion == actual_board {
```

Comment:
I figure there's probably a better way to find the suggestion that corresponds to the move the user actually made, but this is the best I could figure out.

Reply:
I also couldn't decide on a specific location for the config setting related to the suggestion threshold, so I left it as a constant that I can edit. If you'd like me to make this configurable, I'd be more than happy to do so; just let me know where you want it.